### ISSUE TITLE
How to visualize val loss in tensorboard

### ISSUE THREAD

**Issue by Icecream-blue-sky** [open-mmlab/mmdetection#4680]
The tensorboard displayed bbox_mAP of validationset by default, however, what I want is the validation loss.
How could I get that?
Thanks!

**Comment by xvjiarui** [comment]
Hi @Icecream-blue-sky 
You may add `('val', 1)` in to `work_flow`.
